### ISSUE TITLE
Strange 1.16 portal/dimension behavior on servers.

### ISSUE THREAD

**Issue by quek04** [quek04/The-Undergarden#16]
The dimension seems to not register on servers sometimes/entirely unless the host does /reload from the console and then restarts the server. Unknown cause. The dimension not being registered causes the portal to cease function.

**Comment by Eviand25** [comment]
I have the new Undergarden mod installed on my forge server, and I can build the portal and the items all exist on the server, however the portal does not teleport me to the Undergarden. I tried /reload and I stopped the server and turned it back on to no avail.

**Comment by boss101720** [comment]
i can use this in single player but not my server i tried reloading it and nothing :( are you trying to fix it id like to play it with my friend and this is a really awesome mod

**Comment by SovereignSEN** [comment]
I also have the same problem in my multiplayer forge server. Creating the portal works fine, but trying to enter it does not do anything. The /reload and server restart method did not solve the issue unfortunately. However, the portal functions in single player. This is an amazing and beautiful mod that I would like to experience with friends :)

**Comment by JulianPlusPlus** [comment]
This could actually just be a problem with the current version of forge right now. I've heard that this happens to other dimension mods in 1.16, so I think this is out of Quek's control right now. However, I could be wrong about this.

**Comment by darkphan** [comment]
It is easy to validate this (just confirmed by adding another mod with a dimension). Running "/forge dimensions" after first starting a server only shows the vanilla dimensions, but running it after restarting the server shows the vanilla and modded dimensions.

**Comment by quek04** [comment]
going to close this due to the bug not being my fault. it's a forge thing. now please stop asking me to fix it

**Comment by quek04** [comment]
As of Forge 1.16.4-35.0.15 I assume this is fixed.

**Comment by kreezxil** [comment]
the bad behavior with dimension mods is back, I'm on Forge 1.16.5-36.1.4. Works on single player not on server. All of the above statements about building the portal, activating the portal, etc are valid. I have not tried the /reload dimensions thing tho.
Can someone post a version of Forge that this works with for 1.16.5 on a server?

EDIT:

I just tested down to 36.0.46 and no dimension mod works for SMP. I only went that deep as most of the popular mods are compiled against that version of Forge. I'm curious if dimension mods for Fabric work on SMP.
